app/views/schedules/edit.html.erb

```diff
@@ -5,8 +5,9 @@
   <div class="col-md-8 col-md-offset-2">
     <%= form_for(@schedule) do |f| %>
         <%= render partial: 'fields', locals: {f: f, object: @schedule} %>
-        <%= button_tag 'Save', type: :submit, class: 'btn btn-sm btn-primary' %>
-        <%= link_to 'Cancel', schedules_path, class: 'btn btn-sm btn-primary btn-danger' %>
+
+        <%= button_to 'Save', edit_schedule_path(@schedule), class: 'btn btn-sm btn-primary' %>
```

Comment:
I'm not sure what you are saying exactly. Before I changed 'Save' to go to edit_schedule_path, it would render the index upon clicking save. I wanted the user to be able to save tasks and stay on the same page rather than saving, being diverted back to index, then having to click into schedule to edit again. I believe that is effectively the only thing I changed. Am I mistaken?